perception/traffic_light_occlusion_predictor/README.md

```diff
@@ -0,0 +1,34 @@
+# The `traffic_light_occlusion_predictor` Package
+
+## Overview
+
+`traffic_light_occlusion_predictor` receives the detected traffic lights rois and calculates the occlusion ratios of each roi with point cloud.
+
+For each traffic light roi, hundreds of pixels would be selected and projected into the 3D space. Then from the camera point of view, the number of projected pixels that are occluded by the point cloud is counted and used for calculating the occlusion ratio for the roi.
+
+If no point cloud is received or all point clouds have very large stamp difference with the camera image, the occlusion ratio of each roi would be set as 0.
+
```

Comment:
[nits] If you have a simple picture of this description, it would be very helpful for readers

Reply:
I updated the README and add a figure to explain it in commit [692e1e5](https://github.com/autowarefoundation/autoware.universe/pull/4050/commits/692e1e588e0478148fa48b140b55dcbaef4e9186)